CHANGELOG.md

```diff
@@ -5,7 +5,7 @@
 - Removed support for Python 3.5 and added support for 3.7.
 - Moved to `GraphQL-core-next` that supports `async` resolvers, query execution and implements more recent version of GraphQL spec. If you are updating existing project, you will need to uninstall `graphql-core` before installing `graphql-core-next`, as both libraries use `graphql` namespace.
 - Added `gql()` utility that provides GraphQL string validation on declaration time, and enables use of [Apollo-GraphQL](https://marketplace.visualstudio.com/items?itemName=apollographql.vscode-apollo) plugin in Python code.
-- Added `start_simple_server()` shortcut for creating simple server with `GraphQLMiddleware.make_server()` and then running it with `serve_forever()`.
+- Added `start_simple_server()` shortcut that abstracts away dev server creation and start using `GraphQLMiddleware.make_server()`.
```

Comment:
Maybe _a shortcut for_ ?

Reply:
Could you propose how whole line should look like? "shortcut for dev server creation that abstracts away `GraphQLMiddleware.make_server()`"?

Reply:
> Added `start_simple_server()` shortcut function for a quick dev server creation internally using `GraphQLMiddleware.make_server()`.

What do you think @rafalp ?